docs/src/get-started/deploy/commit.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Git commit
+weight: -9
+description: Start getting your project into Platform.sh.
+---
+
+Once you have your project initialized, it's time to add the basics to get it deployed.
+
+Create a file to hold your app configuration:
```

Comment:
```suggestion
In your project, create a file to hold your app configuration:
```

Reply:
I think `project` is somewhat ambiguous here. I'd probably go with `repository`.